src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -210,12 +212,12 @@ private function getAvailableBundles(bool $alias): array
         return $availableBundles;
     }
 
-    private function getConfig(ExtensionInterface $extension, ContainerBuilder $container)
+    private function getConfig(ExtensionInterface $extension, ContainerBuilder $container, bool $resolveEnvs = false)
     {
         return $container->resolveEnvPlaceholders(
             $container->getParameterBag()->resolveValue(
                 $this->getConfigForExtension($extension, $container)
-            )
+            ), true === $resolveEnvs ?: null
```

Comment:
Comparison with `true` is not necessary.
```suggestion
            ), $resolveEnvs ?: null
```

Reply:
Of course, thanks!